draft-fossati-tls-attestation.md

```diff
@@ -129,55 +129,69 @@ attestation.
+attestation technology characteristics, such as evidence encoding format and
+semantics of the evidence contents, are agnostic to the TLS handshake itself.
+Similarly, in the passport model, the characteristics of the attestation results,
+such as encoding format and trust relationships, are agnostic to the TLS
+handshake. Moreover, this specification allows both peers to authenticate
+themselves using remote attestation credentials, and for their attestation
+topologies to be independent of each other.
+
+Several mechanisms for producing and encoding evidence are available, such as:
 
 - the Entity Attestation Token (EAT) {{I-D.ietf-rats-eat}}, 
 - the Trusted Platform Modules (TPMs) {{TPM1.2}} {{TPM2.0}},
 - the Android Key Attestation, and
 - Apple Key Attestation. 
 
 Likewise, there are different encodings available for attestation results. One
-such encoding, AR4SI {{?I-D.ietf-rats-ar4si}} is being standardized by the RATS
+such encoding, AR4SI {{?I-D.ietf-rats-ar4si}}, is being standardized by the RATS
 working group.
 
-This specification supports both the background check and passport model, during
-TLS Handshake. In background check model the details about the attestation
-technology are agnostic to TLS handshake itself. Similarly, in the passport
-model, the details about the attestation results encoding and trust
-relationships are agnostic to the TLS handshake.
+However, this document does not specify how different attestation technologies
+should be defined, only that they should all be usable within our framework.
+Attestation mechanisms must be defined by companion specifications.
 
 To give the peer information that the handshake signing key is properly secured,
 the associated attestation result has to be appraised by the peer. This must be
 the case when either of the two remote attestation topologies is used. Hence,
-the attestation evidence (to be appraised by the verifier) must contain the
-security state of the signing key as well as the overall platform. The platform
-attestation service ensures that the key attestation service has not been
-tampered with. The platform attestation service issues the Platform Attestation
-Token (PAT) and the key attestation service issues the Key Attestation Token
-(KAT). The security of the protocol critically depends on the verifiable binding
-between these two logically separate units of evidence.
-
-This document does not define how different attestation technologies are
-encoded. This is accomplished by companion specifications.
+attestation evidence must contain the security state of both the signing key and
+of the platform hosting it. A Platform Attestation Service issues Platform
+Attestation Tokens (PAT) to prove that the Key Attestation Service has not been
+tampered with. The Key Attestation Service in turn issues Key Attestation Tokens
+(KAT) to prove that the signing key is secure, as described in {{-rats-kat}}. The
```

Comment:
Confused: I thought the rats-kat document is a modular plug-in for the current document, and can be replaced with other implementations. This paragraph sounds like it is mandatory.

Can you please check if this can be consolidated with my proposed text in #32.

Reply:
Yes, this makes sense, I think it would be best if we merge #32 and then I reference that section here!

Reply:
Done. I think you will need to manually merge the text with the current PR.

Reply:
Done!